assets/js/modules/search-console/components/dashboard/SearchFunnelWidget/index.js

```diff
@@ -83,6 +83,35 @@ const SearchFunnelWidget = ( {
 		} )
 	);
 
+	// Get Analytics Goals data
+	const analyticsGoalsData = useSelect( ( select ) =>
+		select( MODULES_ANALYTICS ).getGoals()
+	);
```

Comment:
This will need to be updated to use `useInViewSelect` now that #4121 has been merged 🙂